config/environments/test.rb

```diff
@@ -87,6 +87,7 @@
     Bullet.add_safelist type: :n_plus_one_query, class_name: "User", association: :facilities
     Bullet.add_safelist type: :n_plus_one_query, class_name: "User", association: :price_groups
     Bullet.add_safelist type: :n_plus_one_query, class_name: "User", association: :user_roles
+    Bullet.add_safelist type: :n_plus_one_query, class_name: "EstimateDetail", association: :product
```

Comment:
I think we can preload products for estimate details instead of adding this exception. What do you think?